core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -616,8 +641,18 @@
               disconnected = true;
             },
             onReconnected: function() {
-              bkSessionManager.reconnectEvaluators();
-              disconnected = false;
+              bkSessionManager.isSessionValid()
```

Comment:
So, what well isSessionValid() resolve to if the call to ../beaker/rest/session-backup/getExistingSessions returns a 503?